packages/generator/templates/app/src/app/auth/forgot-password/page.tsx

```diff
@@ -0,0 +1,46 @@
+"use client"
+import {LabeledTextField} from "src/app/components/LabeledTextField"
+import {Form, FORM_ERROR} from "src/app/components/Form"
+import {ForgotPassword} from "../validations"
+import forgotPassword from "../mutations/forgotPassword"
+import {useMutation} from "@blitzjs/rpc"
+import {BlitzPage} from "@blitzjs/next"
+
+const ForgotPasswordPage: BlitzPage = () => {
```

Comment:
```suggestion
export default const ForgotPasswordPage: BlitzPage = () => {
```

```suggestion
export default function ForgotPasswordPage() {
```

The pattern I see in NextJS-Land is to use `export default function ForgotPasswordPage() {}`, eg. https://github.com/vercel/vercel/blob/main/examples/nextjs/app/page.tsx

Its nice and compact and keeps things in one line, so I started using it for page-files.

Reply:
Nice will add it to things to change

Reply:
done